### DISCUSSION TITLE
Get Engine.StringVariables of base setup in upgrades

### DISCUSSION THREAD

**Discussion by kalyani1217** [wixtoolset/issues#6896] [Questions]
Hi, 

I have a base setup of v1.0.0.0. In my base setup, I have set Engine.StringVariables["MYSTRING"]="somevalue".
Now I want to fetch the MYSTRING variable value in the upgrade setup (major or minor release).
I tried to fetch value using Engine.StringVariables["MYSTRING"] but returns the value as null.

Please help with how to fetch base setup engine variables in upgrade setup.

**Comment by rseanhall** [comment]
This feature request is to support what you're asking for - https://github.com/wixtoolset/issues/issues/3704.

Your other option is to write them to the registry yourself like sending them to your MSI package which does something like https://robmensching.com/blog/posts/2010/5/2/the-wix-toolsets-remember-property-pattern/.
